daemon/cmld.c

```diff
@@ -1496,10 +1496,15 @@ cmld_init(const char *path)
 	 * callback (via event_) and parses incoming commands
 	 * and calls the corresponding function in the cmld module,
 	 * e.g. cmld_switch_container */
+
+#ifdef SYSTEMD
+	cmld_control_cml = control_local_new();
+#else
 	if (dir_mkdir_p(CMLD_SOCKET_DIR, 0755) < 0) {
 		FATAL("Could not create directory for cmld_cli control socket");
 	}
 	cmld_control_cml = control_local_new(CMLD_CONTROL_SOCKET);
```

Comment:
We should define CMLD_CONTROL_SOCKET to NULL in Makefile

Reply:
Done. Same for SCD_CONTROL_SOCKET

Reply:
TODO: Fix SCD socket usage in c_smartcard